xml/admin_ceph_upgrade.xml

```diff
@@ -109,24 +111,27 @@
     </listitem>
     <listitem>
      <para>
-      The upgrade from using Nautilus RPMs to Octopus Containers needs to
-      happen all in one step. This means upgrade an entire node at a time, not
-      one daemon at a time.
+      The upgrade from using &prevcephname; RPMs to &cephname; containers needs
+      to happen all in one step. This means upgrade an entire node at a time,
+      not one daemon at a time.
      </para>
     </listitem>
     <listitem>
      <para>
-      The OSD migration from FileStore to Bluestore <emphasis>needs</emphasis>
-      to happen before the upgrade as it is unsupported in &productname;
-      &productnumber;. See <xref linkend="filestore2bluestore"/> for more
-      information.
+      The OSD migration from &filestore; to &bluestore;
+      <emphasis>needs</emphasis> to happen before the upgrade as it is
+      unsupported in &productname; &productnumber;. Find more details about
+      &bluestore; and how to migrate from &filestore; at
+      <link xlink:href="https://documentation.suse.com/ses/6/single-html/ses-deployment/#filestore2bluestore"/>.
      </para>
     </listitem>
     <listitem>
      <para>
       If you are running an older cluster that still uses
       <literal>ceph-disk</literal> OSDs, you <emphasis>need</emphasis> to
-      switch to <literal>ceph-volume</literal> before the upgrade.
+      switch to <literal>ceph-volume</literal> before the upgrade. Find more
```

Comment:
Strictly speaking, it _is_ possible to leave ceph-disk OSDs in place, then switch to ceph-volume after the upgrade by destroying the OSDs and redeploying them. I'm not sure we should necessarily encourage that though.

Reply:
no we should not, that would require me rewrite what i've written there :-)

Reply:
:+1: